dask/array/tests/test_fft.py

```diff
@@ -31,80 +31,52 @@ def test_cant_fft_chunked_axis():
         fft(bad_darr, axis=0)
 
 
-def test_fft():
-    assert_eq(fft(darr), np.fft.fft(nparr))
-    assert_eq(fft(darr2, axis=0), np.fft.fft(nparr, axis=0))
-
-
-def test_fft_n_kwarg():
-    assert_eq(fft(darr, 5), np.fft.fft(nparr, 5))
-    assert_eq(fft(darr, 13), np.fft.fft(nparr, 13))
-    assert_eq(fft(darr2, 5, axis=0), np.fft.fft(nparr, 5, axis=0))
-    assert_eq(fft(darr2, 13, axis=0), np.fft.fft(nparr, 13, axis=0))
-
-
-def test_ifft():
-    assert_eq(ifft(darr), np.fft.ifft(nparr))
-    assert_eq(ifft(darr2, axis=0), np.fft.ifft(nparr, axis=0))
-
-
-def test_ifft_n_kwarg():
-    assert_eq(ifft(darr, 5), np.fft.ifft(nparr, 5))
-    assert_eq(ifft(darr, 13), np.fft.ifft(nparr, 13))
-    assert_eq(ifft(darr2, 5, axis=0), np.fft.ifft(nparr, 5, axis=0))
-    assert_eq(ifft(darr2, 13, axis=0), np.fft.ifft(nparr, 13, axis=0))
-
-
-def test_rfft():
-    assert_eq(rfft(darr), np.fft.rfft(nparr))
-    assert_eq(rfft(darr2, axis=0), np.fft.rfft(nparr, axis=0))
-
-
-def test_rfft_n_kwarg():
-    assert_eq(rfft(darr, 5), np.fft.rfft(nparr, 5))
-    assert_eq(rfft(darr, 13), np.fft.rfft(nparr, 13))
-    assert_eq(rfft(darr2, 5, axis=0), np.fft.rfft(nparr, 5, axis=0))
-    assert_eq(rfft(darr2, 13, axis=0), np.fft.rfft(nparr, 13, axis=0))
-    assert_eq(rfft(darr2, 12, axis=0), np.fft.rfft(nparr, 12, axis=0))
-
-
-def test_irfft():
-    assert_eq(irfft(darr), np.fft.irfft(nparr))
-    assert_eq(irfft(darr2, axis=0), np.fft.irfft(nparr, axis=0))
-
-
-def test_irfft_n_kwarg():
-    assert_eq(irfft(darr, 5), np.fft.irfft(nparr, 5))
-    assert_eq(irfft(darr, 13), np.fft.irfft(nparr, 13))
-    assert_eq(irfft(darr2, 5, axis=0), np.fft.irfft(nparr, 5, axis=0))
-    assert_eq(irfft(darr2, 13, axis=0), np.fft.irfft(nparr, 13, axis=0))
-    assert_eq(irfft(darr2, 12, axis=0), np.fft.irfft(nparr, 12, axis=0))
-
-
-def test_hfft():
-    assert_eq(hfft(darr), np.fft.hfft(nparr))
-    assert_eq(hfft(darr2, axis=0), np.fft.hfft(nparr, axis=0))
-
-
-def test_hfft_nkwarg():
-    assert_eq(hfft(darr, 5), np.fft.hfft(nparr, 5))
-    assert_eq(hfft(darr, 13), np.fft.hfft(nparr, 13))
-    assert_eq(hfft(darr2, 5, axis=0), np.fft.hfft(nparr, 5, axis=0))
-    assert_eq(hfft(darr2, 13, axis=0), np.fft.hfft(nparr, 13, axis=0))
-    assert_eq(hfft(darr2, 12, axis=0), np.fft.hfft(nparr, 12, axis=0))
-
+@pytest.mark.parametrize(
+    "funcname",
+    [
+        "fft",
+        "ifft",
+        "rfft",
+        "irfft",
+        "hfft",
+        "ihfft",
+    ]
+)
+def test_fft(funcname):
+    da_fft = getattr(da.fft, funcname)
+    np_fft = getattr(np.fft, funcname)
 
-def test_ihfft():
-    assert_eq(ihfft(darr), np.fft.ihfft(nparr))
-    assert_eq(ihfft(darr2, axis=0), np.fft.ihfft(nparr, axis=0))
+    assert_eq(da_fft(darr),
+              np_fft(nparr))
```

Comment:
Sorry, I meant to say, is it possible to merge this entire `test_fft` with the test below?  Perhaps something like

```python
@pytest.mark.parametrize('axis', [-1, 0])
@pytest.mark.parametrize('n', [0, 5, 13])
@pytest.mark.paramtrize('funcname', ...)
def test_fft(funcname, axis, n):
    da_fft = getattr(da.fft, funcname)
    np_fft = getattr(np.fft, funcname)
    assert_eq(da_fft(darr, n=n, axis=axis), 
              np_fft(nparr, n=n, axis=axis)
```